sharepoint/sharepoint-ps/sharepoint-online/Get-SPOContainer.md

```diff
@@ -37,13 +37,13 @@ Get-SPOContainer [-OwningApplicationId <OwningApplicationId>] [-Paged] [-PagingT
 ### ParamSet4
 
 ```powershell
-Get-SPOContainer [-OwningApplicationId <OwningApplicationId>] [[-Identity] <ContainerId>]
```

Comment:
>-OwningApplicationId <OwningApplicationId>] [](http://example.com/codeflow?start=18&length=43)

the cmdlet code itself still has this OwningApplication param. Will it be removed from there too?

Reply:
Sam, the cmdlet code itself is already updated and it is in prod. The documents were not.